pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -326,38 +339,101 @@ func indentLines(indent string, lines ...string) []string {
 	return ret
 }
 
-func describeProjectAndServer(project *projectapi.Project, server string) string {
-	if server != "" {
+type formatter interface {
+	ResourceName(obj interface{}) string
+}
+
+func namespaceNameWithType(resource, name, namespace, defaultNamespace string, noNamespace bool) string {
+	if noNamespace || namespace == defaultNamespace || len(namespace) == 0 {
+		return resource + "/" + name
+	}
+	return resource + "/" + name + "[" + namespace + "]"
```

Comment:
Talked with @liggitt, I think we should do `resource.group[namespace]/name`.  Doing it that way makes 

```bash
oc get resource
oc get resource.group
oc get resource[namespace]
oc get resource.group[namespace]

oc get resource/name
oc get resource.group/name
oc get resource[namespace]/name
oc get resource.group[namespace]/name
```

parse nicely and when sorting, items of a given resource type are grouped by namespace instead of by name.